cascade layer and cascade origin (close #425) (#453)
web-standards-ru/dictionary: dictionary.md

### cascade

> **каскад,** описанная в спецификации CSS схема, определяющая порядок применения к элементу блоков стилей.
> 
> ### cascade layer
> 
> Точно так же, как [каскадные источники](#cascade-origin) обеспечивают баланс между стилями пользователя и разработчика, **каскадные слои** обеспечивают способ организации в рамках одного источника. Объявляются с помощью [директивы](#at-rule) `@layer`.
> 
> ### cascade origin
> 
> **каскадные источники** - уровни, на которых возможно объявление стилей в css. Существует 8 уровней: самый высокий - transition-декларации, самый низкий - декларации user-агента.

Reviewer: Предлагаю немного формализовать статьи для отдельных терминов: единстевнное число, единый формат, примеры.

```suggestion
### cascade

**каскад,** описанная в спецификации CSS схема, определяющая порядок применения к элементу блоков стилей.

### cascade layer

**каскадный слой,** способ группировки и управления порядком применения стилей в рамках одного [источника](#cascade-origin), объявляется с помощью [директивы](#at-rule) `@layer`. Напр. _rules within a cascade layer cascade together_ — _правила в каскадных слоях каскадируют вместе._

### cascade origin

**каскадный источник,** встроенная в браузеры область описания стилей, их порядок определяет применение стилей или [каскад](#cascade): от стилей браузера до transition-деклараций. Напр. _there are eight cascade origings, including user styles_ — _есть восемь каскадных источников, включая пользовательские стили._
```

### cascade

### cascade

**каскад,** описанная в спецификации CSS схема, определяющая порядок применения к элементу блоков стилей.

### cascade layer

**каскадный слой,** способ группировки и управления порядком применения стилей в рамках одного [источника](#cascade-origin), объявляется с помощью [директивы](#at-rule) `@layer`. Напр. _rules within a cascade layer cascade together_ — _правила в каскадных слоях каскадируют вместе._

### cascade origin

> **каскадный источник,** встроенная в браузеры область описания стилей, их порядок определяет применение стилей или [каскад](#cascade): от стилей браузера до transition-деклараций. Напр. _there are eight cascade origings, including user styles_ — _есть восемь каскадных источников, включая пользовательские стили._

Reviewer: ```suggestion
**каскадный источник,** встроенная в браузеры область описания стилей, порядок источников определяет применение стилей или [каскад](#cascade): от стилей браузера до transition-деклараций. Напр. _there are eight cascade origings, including user styles_ — _есть восемь каскадных источников, включая пользовательские стили._
```

Reviewer: @skip405 так лучше?

**каскад,** описанная в спецификации CSS схема, определяющая порядок применения к элементу блоков стилей — в зависимости от их источника, важности, [специфичности](#specificity) и положения в коде. Ошибочно «каскадом» иногда называют [селектор потомка](#selector).

> ### cascade layer
> 
> Точно так же, как [каскадные источники](#cascade-origin) обеспечивают баланс между стилями пользователя и разработчика, **каскадные слои** обеспечивают способ организации структуры в рамках одного источника. Объявляются с помощью [директивы](#at-rule) `@layer`.
> 
> ### cascade origin
> 
> **каскадные источники** - уровни, на которых возможно объявление стилей в css. Существует 8 уровней: самый высокий - transition-декларации, самый низкий - декларации user-агента.
> 

Reviewer: ```suggestion

- cascade layers — **каскадные слои**
- cascade origins — **каскадные источники**
```

Мне кажется, что важно не делать из словаря справочник. Главная задача здесь — не описать термин, а зафиксировать перевод и написание. Суть термина уже описана в переводе слова «каскад», дальше можно дать варианты использования. Есть такой паттерн в других частях словаря.

Если кажется, что описание каскада устарело и к слоям и источникам хорошо бы его подновить — велком :)

Author: Я принесла "каскадные слои", потому что у меня возник спор на тему слои это, или уровни, или что-то другое. А "каскадные источники" потому, что через них удобно объяснить, что такое слои.

Reviewer: Это очень полезное добавление в словарь! Просто тут нет задачи объяснять термины, скорее фиксировать перевод, чтобы было проще писать, переводить и даже говорить. Если хочется объяснить что это такое по-русски, то кажется лучше зайти, скажем, [в Доку](https://doka.guide/) и сделать там простую вводную доку. [Вот, например, про `@media`](https://doka.guide/css/media/).
